department-of-veterans-affairs/abd-vro
# Feature - change lookup table mappings

## What was the problem?

After discussion in the benefits-cft-classification slack, we have come to the understanding that because MAS bot will likely overwrite many of our classifications, it is better to start mapping diagnostic codes for classifications that are *not* automation eligible.

Associated tickets or Slack threads:

- "Build a table for Yurena's lookup table" for https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/160

## How does this fix it?[^1]

- This PR updates the lookup table mapping to use the codes for non-automation eligible classifications available [here](https://docs.google.com/spreadsheets/d/18Mwnn9-cvJIRRupQyQ2zLYOBm3bd0pr4kKlsZtFiyc0/edit#gid=1711756762).
- The underlying API code does not change

## How to test this PR
- navigate to `domain-cc/svc-cc-app`, run `pytest`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
